.github/workflows/rtd.yml

```diff
@@ -0,0 +1,21 @@
+name: Update readthedocs branch
+
+on:
+  push:
+    branches: ["branch-0.20"]
+
+jobs:
+  update:
+    name: Update readthedocs branch
+    runs-on: "ubuntu-latest"
+    steps:
+      - name: Checkout readthedocs
+        uses: actions/checkout@v2.3.4
+        with:
+          ref: readthedocs
```

Comment:
Are we okay with this name for the branch? Regardless of the name, we'll need to make a new branch for this that uses the no libnuma version of UCX, as shown in:

- https://github.com/quasiben/ucx-py/blob/fix-rtd/conda/environments/builddocs_py37.yml

Reply:
I'm fine with that name. Do you have permission to push a new branch? If not I can try it (not sure if I have permission as well).

Reply:
I don't have permission - is this limited to `ucxpy-python-codeowners` or some other group?

Reply:
I see the new `readthedocs` branch, but it's not an empty branch. Is that intended? If not, I can re-create it as an empty branch.

Reply:
That's intended. The `readthedocs` branch is a fork of `branch-0.20` with additional changes to https://github.com/rapidsai/ucx-py/blob/readthedocs/conda/environments/builddocs_py37.yml, specifically the changes we need are in https://github.com/rapidsai/ucx-py/commit/8535c9c4e4452bd51e3b57f1ccd72fdf41d936f2 , which gets a UCX build that's not dependent on `libnuma`. From this point on, every time a PR is merged into `branch-0.20`, the changes from the present PR will be merged into that and RTD rebuild and published.